src/components/Button.js

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 
-export default function Button({ name, task, onDelete }) {
+export default function Button({ name, id, onDelete }) {
     return (
         <>
             {name === "Submit" && <button className='btn btn-submit'>{name}</button>}
-            {name === "X" &&  <button className='btn btn-close' onClick={() => onDelete(task.id)}>{name}</button>}
+            {name === "X" &&  <button className='btn btn-close' onClick={() => onDelete(id)}>{name}</button>}
             {name === "<" && <button className='btn btn-prev-next'>{name}</button>}
             {name === ">" && <button className='btn btn-prev-next'>{name}</button>}
```

Comment:
do you think it will be better to use a `switch` statement?
Will make it easier to read.

ex: 
```
function Button () {
  function getButton () {
    switch (name){}
  }
  
  return (
    {getButton()}
  )
}
```

Reply:
I think so. I will fix it.